packages/svelte/tests/css/samples/class-directive/input.svelte

```diff
@@ -1,7 +1,8 @@
-<div class:first={true} class:second={true}></div>
+<div class="zero" class:first={true} class:second={true}></div>
```

Comment:
Can you adjust it to this? Tests the "nothing beforehand" case, too.
```suggestion
<div class:first={true} class:second={true}></div>
<div class="zero" class:first={true} class:second={true}></div>
```